packages/ckeditor5-core/src/editor/editor.ts

```diff
@@ -527,10 +527,16 @@ export default abstract class Editor extends /* #__PURE__ */ ObservableMixin() {
 
 			if ( [ 'evaluation', 'trial' ].includes( licensePayload.licenseType ) ) {
 				const licenseType: 'evaluation' | 'trial' = licensePayload.licenseType;
+				const capitalizedLicenseType = licenseType[ 0 ].toUpperCase() + licenseType.slice( 1 );
 
 				console.info(
-					`You are using the ${ licenseType } version of CKEditor 5 with limited usage. ` +
-					'Make sure you will not use it in the production environment.'
+					`%cCKEditor 5 ${ capitalizedLicenseType } License`,
```

Comment:
20 lines below, we have a very similar text.

Reply:
Fixed in later commits